### PR TITLE
seqence length and path fix.

### PR DESCRIPTION
**Pull request by alchemistcai**
In `get_esm_embedding.py`, result['mask'] raises KeyError.In my test,result only has `logits,representations` keys.I use esm's official example to calculate sequence length and it works.

In `data_process.py`,max_repr_esm = np.load("multi_modal_binding/data/normalization_constants/esm_repr_min.npy")` can't locate the file.I change the path to abspath and it works.

```py
    current_dir=os.path.dirname(__file__)
    max_repr_esm_if = np.load(
        os.path.join(current_dir,"normalization_constants/esm_if_repr_max.npy")
    )
```

### CODE REVIEW

**Comment by alchemistcai** [comment]
Other issues I encontered and my solutions.
